### DISCUSSION TITLE
Feature request: SAVE button visible while preview is open

### DISCUSSION THREAD

**Discussion by bencresty** [craftcms/cms#10066] [Ideas] [answered]
Hi,

I love the live preview of Craft. It's pretty great creating blog posts with the preview open so we can see the result immediately.

The experience could be even better if the SAVE button would be on the edit panel while the preview panel is open. As right now we have to close the preview panel every time we want to save.

Also it would be nice if ctrl-S to save and continue editing would keep the preview open when used from inside the 'preview mode'.

**Comment by brandonkelly** [comment, marked as answer]
This was previously requested by #9010.

**Reply by bencresty** [reply]
@brandonkelly  Thanks. I'll plus that one then
